packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -226,7 +226,7 @@ def validate_volume_limits(cls, v, info: ValidationInfo) -> str | None:
                     "outputs_path": "/tmp/outputs",  # noqa: S108 nosec
                     "inputs_path": "/tmp/inputs",  # noqa: S108 nosec
                     "state_paths": ["/tmp/save_1", "/tmp_save_2"],  # noqa: S108 nosec
-                    "state_exclude": ["/tmp/strip_me/*", "*.py"],  # noqa: S108 nosec
+                    "state_exclude": ["/tmp/strip_me/*"],  # noqa: S108 nosec
```

Comment:
why removing that one?

Reply:
To fix a flaky test. Just realised the proper fix. the `state_exclude` is a set. Each time we serialise it we get a list. Have fun figuring out which is the correct order of the elements.
If you pass via the set, depending on the current hash see it might be as you expect or with the elements inverted.